Modules/Core/Common/include/itkThreadSafeBooleanType.h

```diff
@@ -0,0 +1,63 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkThreadSafeBooleanType_h
+#define itkThreadSafeBooleanType_h
+
+namespace itk
+{
+
+/** The type alias `ThreadSafeBooleanType` provides an alternative to `bool`. `std::vector<bool>` is not thread safe due
```

Comment:
Actually it's not a type alias. It's a `class`! So maybe it should not end with `Type` anyway. I think in ITK, the postfix `Type` is (usually) only used for typedef's (type aliases), and `using` declarations thereof.

Reply:
Will fix.